web/styles/app_variables.css

```diff
@@ -486,8 +487,12 @@
     --color-background-tab-picker-selected-tab: hsl(0deg 0% 100% / 7%);
     --color-outline-tab-picker-tab-option: hsl(0deg 0% 100% / 12%);
     --color-background-tab-picker-tab-option-hover: hsl(0deg 0% 100% / 5%);
-    --color-background-alert-word: hsl(22deg 70% 35%);
     --color-buddy-list-highlighted-user: hsl(136deg 25% 73% / 20%);
+    --color-background-alert-word: hsl(18deg 100% 75%);
```

Comment:
Do future contributors a favor, and make sure that the order in the light theme is preserved just as it is here, if you're going to reorder variables.